apps/work/filters.py

```diff
@@ -0,0 +1,21 @@
+import strawberry
+from typing import List
+
+from .models import Work
+from .enums import CategoryEnum, TagEnum
+
+
+@strawberry.django.filters.filter(Work)
+class WorkFilter:
+    category: List[CategoryEnum] | None
+    tag: List[TagEnum] | None
```

Comment:
Newer python has direct support for typing for basic types. No need to import List from typing.
```suggestion
    category: list[CategoryEnum] | None
    tag: list[TagEnum] | None
```